libs/ts/decoders/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@blocksense/decoders",
+  "scripts": {
+    "clean": "git clean -fdx -e .env",
+    "build": "hardhat compile && yarn build:typecheck",
+    "test": "hardhat test",
+    "test:paris": "EVM_VERSION=paris yarn hardhat test",
+    "ts": "yarn node --import tsx --no-deprecation",
+    "deploy:local": "hardhat run --network localhost"
+  },
+  "devDependencies": {
+    "@blocksense/base-utils": "workspace:*",
+    "@blocksense/contracts": "workspace:*",
+    "@chainsafe/ssz": "^1.0.2",
+    "@nomicfoundation/hardhat-chai-matchers": "^2.1.0",
+    "@nomicfoundation/hardhat-ethers": "^3.1.0",
+    "@nomicfoundation/hardhat-network-helpers": "^1.1.0",
+    "@nomicfoundation/hardhat-toolbox": "^5.0.0",
+    "@typechain/ethers-v6": "^0.5.1",
+    "@typechain/hardhat": "^9.1.0",
+    "@types/chai": "4",
+    "@types/ejs": "^3",
+    "@types/mocha": "^10.0.10",
+    "@types/node": "^24.3.1",
+    "chai": "4",
+    "dotenv": "^16.4.7",
```

Comment:
```suggestion
```

Direnv handles loading of `.env` files for us [here](https://github.com/blocksense-network/blocksense/blob/7a0250fc0da849b12b7c47f78fae4e7d362f5a28/.envrc#L9).